specs/005-structured-logging/plan.md

```
@@ -0,0 +1,64 @@
+### HASH_SALT の要件
+
+- 最低 32 バイト以上のランダム値
+- 環境変数で管理
+- 本番環境では必須
+
+## 検証項目
+
+- [x] ローカル環境でログ出力を確認
+- [x] ユニットテストが通過
+- [x] 型チェックが通過
+- [x] lint が通過
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)